SoftLayer/fixtures/SoftLayer_Location.py

```diff
@@ -9,3 +9,4 @@
         "longName": "San Jose 1",
         "name": "sjc01"
     }]
+getDatacenters = [{'id': 1854895, 'name': 'dal13', 'regions': [{'keyname': 'DALLAS13'}]}]
```

Comment:
Might be good to combine the fixtures since they are for the same method.

Reply:
The problem is that my method expects only 1 result back, but else where expects multiple results back.... I need to figure out a better way to do these fixtures to deal with different object masks/filters. but that might be for a different project.